Solutions/VaronisSaaS/Data Connectors/VaronisSaaS_API_FunctionApp.json

```diff
@@ -0,0 +1,127 @@
+{
+  "id": "VaronisSaaS",
+  "title": "Varonis SaaS",
+  "publisher": "Varonis",
+  "descriptionMarkdown": "Varonis SaaS provides the capability to ingest [Varonis Alerts](https://www.varonis.com/products/datalert) into Microsoft Sentinel.",
```

Comment:
Also add a few more lines to Description markdown of Data connector

Reply:
I'm waiting for copywriter, and I will update as soon as I have text.

Reply:
on this one